src/category_theory/limits/cones.lean

```diff
@@ -133,6 +134,18 @@ def whisker {K : Type v} [small_category K] (E : K ⥤ J) (c : cone F) : cone (E
 
 @[simp] lemma whisker_π_app (c : cone F) {K : Type v} [small_category K] (E : K ⥤ J) (k : K) :
   (c.whisker E).π.app k = (c.π).app (E.obj k) := rfl
+
+section
+omit 𝒞
+variables {m : Type v → Type v}
+variables (hom : ∀ {α β : Type v}, m α → m β → (α → β) → Prop)
+variables [h : concrete_category @hom]
+include h
+
+@[simp] lemma naturality_bundled {G : J ⥤ bundled m} (s : cone G) {j j' : J} (f : j ⟶ j') (x : s.X) :
```

Comment:
Shouldn't this move to the file `concrete_category.lean`? (Idem downstairs.)

Reply:
I don't think so. This would make `concrete_category.lean` depend on `limits/cones.lean`, which seems a bad idea to me.

Reply:
Fair enough.